tutorials/axios/axios-data.md

```diff
@@ -0,0 +1,21 @@
+When making a standard axios request, the response contains six properties including `data`, the one we will be discussing. `data` is simply the response that was provided by the server. `data` could be anything, it could be a string, number, or object depending on what information you requested from the server. In the example below, data is an object as it contains a series of key value pairs.
```

Comment:
Normally, `data` is either string, object, or `undefined`. But advanced users can configure it using the [`responseType` option](https://github.com/axios/axios#request-config).

Reply:
Also, the more correct way to describe `res.data` is that it contains the HTTP response body.